projects/wave-core-new/src/lib/layers/layer-list/layer-list-menu/layer-list-menu.component.scss

```diff
@@ -0,0 +1,46 @@
+.container {
+    max-height: inherit;
+    height: inherit;
+    border-right: solid 1px #ccc;
```

Comment:
i  think this needs to go

Reply:
I removed the borders 49a729f5300f16b060c80af4725e1037e8d29681